extensions/wikia/CreateNewWiki/css/CreateNewWiki.scss

```diff
@@ -150,8 +150,9 @@
 
 	.error-msg {
 		color: $wds-color-alert;
-		float: left;
 		font-size: $wds-typescale-size-minus-2;
+		position: absolute;
```

Comment:
It's possible, but the whole content beneath the error would be moving down when the error is displayed. For the same reason it was using float.